### ISSUE TITLE
migrations Table not found

### ISSUE THREAD

**Issue by Gummibeer** [thekordy/ticketit#93]
```
[Illuminate\Database\QueryException]                                         
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'migrations' doesn't exist (SQL: select * from migrations)
```

This error occurs if I try to migrate without a migration table - this method `inactiveMigrations()` in the `InstallerController` is the reason.

Why checking the Migration state? The `artisan migrate`command does this already.

**Comment by thekordy** [comment]
Disable the TicketitServiceProvider, install the migration table, install users and passwords tables, and then install Ticketit. They are pre-requirements

**Comment by nCrazed** [comment]
@thekordy is there any risk associated with enabling the ticketit service provider only for the HTTP Kernel and skipping it in the CLI Kernel?

**Comment by nCrazed** [comment]
Ok, looks like the issue has been addressed in 0.2.5
